docs/sources/datasources/pyroscope/_index.md

```diff
@@ -0,0 +1,80 @@
+---
+aliases:
+  - ../features/datasources/phlare/
+  - ../features/datasources/grafana-pyroscope/
+  - ../datasources/grafana-pyroscope/
```

Comment:
To make it easier to track the intended source of the redirect in the future

```suggestion
  - ../features/datasources/phlare/ # /docs/grafana/<GRAFANA_VERSION>/features/datasources/phlare/
  - ../features/datasources/grafana-pyroscope/ # /docs/grafana/<GRAFANA_VERSION>/features/datasources/grafana-pyroscope/
  - ../datasources/grafana-pyroscope/ # /docs/grafana/<GRAFANA_VERSION>/datasources/grafana-pyroscope/
```

Reply:
Thanks, Jack. Aliases are one thing that are tricky to get right.